src/components/DeleteModal.jsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components'
+import { useState } from 'react'
+import { Modal } from './Modal'
+import { Text } from './Text'
+import { useTranslation } from 'react-i18next'
+import { Button } from './buttons/Button'
+
+export const DeleteModal = ({ visible, setVisible, deleteFn, text, title, setIsAskingAgain }) => {
+  const { t } = useTranslation()
+  const [isChecked, setIsChecked] = useState(false)
```

Comment:
You don't need this state is't duplication because you already have parent state for this checkbox